net/acme/files/acme.config

```diff
@@ -8,5 +8,6 @@ config cert 'example'
        option use_staging 1
        option keylength 2048
        option update_uhttpd 1
+	   option update_nginx 1
```

Comment:
Please use the same whitespace for indentation as the rest of the script (there are loads of these errors; please fix all of them).